packages/next/src/server/app-render/app-render.tsx

```diff
@@ -911,11 +923,11 @@ async function getErrorRSCPayload(
 
   // For metadata notFound error there's no global not found boundary on top
   // so we create a not found page with AppRouter
-  const initialSeedData: CacheNodeSeedData = [
+  const seedData: CacheNodeSeedData = [
     initialTree[0],
     <html id="__next_error__">
-      <head></head>
-      <body></body>
+      <head>{initialHeadMetadata}</head>
+      <body />
```

Comment:
I don't think `body` tags are meant to be self-closing, was this intentional?

Reply:
They're the same as it's just `jsx('body')` with children `null`. Just simplify it along the way while refactoring.